.github/workflows/update-ic.yml

```diff
@@ -0,0 +1,102 @@
+# A GitHub Actions workflow that regularly makes a PR to update
+# the IC candid and code bindings.
+name: Update IC
+on:
+  schedule:
+    # Checks for new IC commit weekly
+    - cron: '30 3 * * MON'
+  workflow_dispatch:
+    # Provides an option to run this manually
+    ic_ref:
+      description: "The IC commit or tag to update to."
+      required: false
+  push:
+    branches:
+      # Runs if there is a change to the development branch for this workflow
+      - "update-ic"
+jobs:
+  update-ic:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ic-js
+        uses: actions/checkout@v3
+      - name: Choose IC ref
+        id: choose_ic_ref
+        run: |
+           ref="${{ inputs.ic_ref }}"
+           if [[ "${requested_ref:-}" == "" ]]
+           then # Use the latest release
+                ref="$(gh release --repo dfinity/ic view --json tagName -q .tagName)"
+           fi
+           echo "ic_ref=$ref" >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ github.token }}
+      - name: Checkout ic repo
+        uses: actions/checkout@v3
+        with:
+          repository: dfinity/ic
+          ref: ${{ steps.choose_ic_ref.outputs.ic_ref }}
+          path: ic
+      - name: Check new IC commit
```

Comment:
Can you rename this step?

How about "Update candid and related JS and TS files"